crates/cubecl-runtime/tests/integration_test.rs

```diff
@@ -250,8 +250,8 @@ fn autotune_cache_different_keys_return_a_cache_miss() {
         dummy::CacheTestAutotuneOperationSet::new(client.clone(), shapes_1, handles_1);
     let cache_test_autotune_kernel_2 =
         dummy::CacheTestAutotuneOperationSet::new(client.clone(), shapes_2, handles_2);
-    autotune_execute(&client, Box::new(cache_test_autotune_kernel_1));
-    autotune_execute(&client, Box::new(cache_test_autotune_kernel_2));
+    autotune_execute(&client, Arc::new(cache_test_autotune_kernel_1));
```

Comment:
The reason to use a `Box` was so that tensors stored in the autotune operation set have their ownership intact to allow for inplace operations. But I don't think it matters to compute benchmarks, and in fact probably contributed to more variance (we still cloned the tensors because we run 10 samples).

Reply:
Hmm I see yeah, tricky. Don't think there's a good way around this as long as the set needs to be shared with another thread (/ future). I guess in the future it'd be possible to run the benchmark on something like Tensor::empty_like(a) if the tensor seems to be possible to in-place otherwise.